### DISCUSSION TITLE
confused getting started (config?)

### DISCUSSION THREAD

**Discussion by gejoreni** [briis/unifiprotect#254] [Q&A]
I was able to follow the setup and get HACS installed - I got unifi protect installed. The place I'm getting hung up is super basic:

"add your UniFi Protect server by providing the Host IP, Port Number, Username and Password"

I don't see where I add that information it did not ask me for it when I installed and there doesn't seem to be a place for it. 

I'm pretty new to all this - sorry if this is a dumb question.

**Comment by briis** [comment]
So if you have installed the Integration via HACS, and restarted Home Assistant, then:
* go to *Configuration* -> *Integrations*. 
* Once you are on this page, do a hard refresh of your browser to ensure the cache is cleared. 
* Then click the '+ ADD INTEGRATION`in the lower right corner, and in here you search for Unifi Protect. 
* You should then get the box where you can add your credentials.
